genshin/utility/logfile.py

```diff
@@ -11,15 +11,34 @@
 PathLike = typing.Union[str, pathlib.Path]
 
 AUTHKEY_FILE = fs.get_tempdir() / "genshin_authkey.txt"
+GAME_LOCATION = fs.get_tempdir() / "genshin_location.txt"
```

Comment:
Instead of saving game location in a file, it would probably be better to keep using `get_logfile` like before except with different possible paths. A path could still be given to `get_authkey()`, so nothing changes there.